CMakeLists.txt

```diff
@@ -973,25 +973,32 @@ if (BUILD_CLIENT)
     set(SHADERS_CPP_TEXT "// This file is auto-generated by CMakeLists.txt.\n")
     string(APPEND SHADERS_CPP_TEXT "#include \"common/Common.h\"\n\n")
     set(SHADERMAP_TEXT "")
+
     foreach(res ${GLSLSOURCELIST})
         get_filename_component(filename_no_ext ${res} NAME_WE)
         set(outpath ${EMBED_INCLUDE_DIR}/${filename_no_ext}.glsl.h)
+        list(APPEND TARGET_client_FILES ${outpath})
```

Comment:
What's this?

Reply:
Something I forgot to delete. Now removed.